instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/TracingExecutionInterceptor.java

```diff
@@ -89,6 +90,14 @@ public SdkRequest modifyRequest(
 
     io.opentelemetry.context.Context parentOtelContext = io.opentelemetry.context.Context.current();
     SdkRequest request = context.request();
+
+    // Ignore presign request. These requests don't run all interceptor methods and the span created
+    // here would never be ended and scope closed.
+    if (executionAttributes.getAttribute(AwsSignerExecutionAttribute.SERVICE_SIGNING_NAME)
```

Comment:
The assumption here is that nothing besides https://github.com/aws/aws-sdk-java-v2/blob/fcaf70fa89a7dccea210435488365f0a1ca1b609/services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/signing/DefaultS3Presigner.java#L332 set that attribute